src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchElementViewModel.cs

```diff
@@ -104,6 +104,23 @@ public void OnRequestDownload(PackageVersion version, bool downloadToCustomPath)
                 RequestDownload(this.Model, version, downloadPath);
         }
 
+        /// <summary>
+        /// Comparator 
+        /// </summary>
+        /// <param name="other">The PackageManagerSearchElementViewModel to compare</param>
+        /// <returns></returns>
+        public bool Equals(PackageManagerSearchElementViewModel other)
+        {
+            if (other == null) return false;
+            return this.Model.Id == other.Model.Id && this.Model.Name == other.Model.Name;
+        }
+
+        /// <summary>
+        /// Overridden Getter for HashCode 
+        /// </summary>
+        /// <returns>HashCode of package</returns>
+        public override int GetHashCode() => (Model.Id, Model.Name).GetHashCode();
```

Comment:
What does this translate to? Not familiar with this syntax. C#?

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7#tuples

Reply:
so this will end up doing this:
https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/Tuple.cs#L264

and combine the hashcodes of the items.

Reply:
@aparajit-pratap @mjkkirschner Since we can't fix the VS version in image easily, I will revert my C# 7 syntax